modules/react/checkbox/lib/CheckboxInput.tsx

```diff
@@ -78,28 +78,28 @@ const checkboxInputStencil = createStencil({
     // Disabled State
     '&:not(:where(:checked, :indeterminate, :disabled, :focus-visible, .focus)):where(:hover, .hover, :active, .active) ~ div:first-of-type':
       {
-        borderColor: base.licorice500,
+        borderColor: system.color.border.input.strong,
       },
     '&:where(:checked, :indeterminate) ~ div:first-of-type': {
       borderColor: brand.primary.base,
       backgroundColor: brand.primary.base,
     },
     '&:disabled ~ div:first-of-type': {
-      borderColor: base.licorice100,
-      backgroundColor: base.soap100,
-      opacity: 1,
+      borderColor: system.color.border.input.disabled,
+      backgroundColor: system.color.bg.alt.softer,
+      opacity: system.opacity.full,
     },
     '&:disabled:where(:checked, :indeterminate) ~ div:first-of-type': {
-      borderColor: brand.primary.light,
-      backgroundColor: brand.primary.light,
+      borderColor: cssVar(brand.primary.light, base.blueberry200),
```

Comment:
There is no `border` color for `blueberry200`, so I keep a base color for fallback.